### PR TITLE
Use latest CRI-O stable as docs default

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
We should use the latest stable version to hint end users to the right stream.

#### Which issue(s) this PR fixes:

Refers to https://github.com/cri-o/cri-o/issues/8353

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,15 +92,15 @@ All packages are based on the static binary bundles provided by the CRI-O CI.
...
 [![release-1.28](https://img.shields.io/badge/prerelease-release--1.28-blue?logo=git&logoColor=white)](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:prerelease:v1.28)
 
-### Distributions using `rpm` packages
-
 #### Define the Kubernetes version and used CRI-O stream
 
 ```bash
 KUBERNETES_VERSION=v1.30
-PROJECT_PATH=prerelease:/main
+CRIO_STREAM=stable:/v1.30
```
Comment (afbjorklund) [inline README.md:99]: Better to use "v1.30" here, and stable:/ below ?
Or use the full project path for both of them

Comment (afbjorklund) [reply README.md:99]: Just thinking that since they use similar OBS setups, the variables and URL templates should be similar too

Comment (saschagrunert) [reply README.md]: Yes, I'm changing the path and set it to `CRIO_VERSION=v1.30`.

**Comment by saschagrunert** [comment]
@cri-o/cri-o-maintainers PTAL

**Comment by kwilczynski** [comment]
@afbjorklund, thank you! :tada:

**Comment by afbjorklund** [comment]
No worries, now it just needs to be propagated to the home page as well...

https://github.com/cri-o/cri-o.io/blob/master/index.md

`https://pkgs.k8s.io/core:/stable:/v1.28`

`https://pkgs.k8s.io/addons:/cri-o:/prerelease:/main`

**Pull request merged**